src/browser/BrowserShared.cpp

```diff
@@ -38,14 +38,15 @@ namespace BrowserShared
 
         // Put the socket in a dedicated directory.
         // This directory will be easily mountable by sandbox containers.
-        QString subPath = path + "/app/org.keepassxc.KeePassXC/";
+        QString subPath = path + "/app/org.keepassxc.KeePassXC";
```

Comment:
Why do we actually prefix the subdirectory path with `/app`?

Reply:
it's apparently a standard, this came out of a flatpak discussion... hence why flatpak only uses /app/{appid} path. The reason was also for apparmor in general, can allow access to this subdir without exposing the entire run directory and not being specific to keepassxc.

Reply:
But is the app part of it also a "standard"?

Reply:
from what I can tell, yes

Reply:
The only app subfolder I have in /run/user/1000 is ours.

Reply:
I'm really unconcerned and don't want to change this again

Reply:
It's not very important, it just appears strange to me. I believe the only reason why there is "app" is to avoid clashes with our previous symlink, which is kind of irrelevant.